src/pages/Login/useLoginForm.ts

```diff
@@ -0,0 +1,56 @@
+import { useState } from 'react'
+
+export function useLoginForm() {
+  const [email, setEmail] = useState('')
+  const [emailError, setEmailError] = useState('')
+
+  const [password, setPassword] = useState('')
+  const [passwordError, setPasswordError] = useState('')
+
+  const validateEmail = (value: string) => {
+    if (!value.trim()) return 'ID를 입력해주세요.'
+    const emailRegex = /^[^\s@]+@[^\s@]+\.[^\s@]+$/
+    if (!emailRegex.test(value)) return 'ID는 이메일 형식으로 입력해주세요.'
+    return ''
+  }
+
+  const validatePassword = (value: string) => {
+    if (!value.trim()) return 'PW를 입력해주세요.'
+    if (value.length < 8) return 'PW는 최소 8글자 이상이어야 합니다.'
+    return ''
+  }
+
+  const handleEmailChange = (e: React.ChangeEvent<HTMLInputElement>) => {
```

Comment:
여기서 꺼내서 사용하는 방법도 있을거 같네요 
```suggestion
  const handleEmailChange = ({ target: { value }}: React.ChangeEvent<HTMLInputElement>) => {
```